repository_files.go

```diff
@@ -43,6 +43,7 @@ type File struct {
 	Ref      string `json:"ref"`
 	BlobID   string `json:"blob_id"`
 	CommitID string `json:"commit_id"`
+	Sha256   string `json:"content_sha256"`
```

Comment:
This should be `SHA` in Go (where abbreviations are in all caps).

Reply:
Oh yes! Thanks, fixed that